sdk/storage/Azure.Storage.Common/src/Azure.Storage.Common.csproj

```diff
@@ -30,6 +30,7 @@
   <ItemGroup>
     <Compile Remove="Shared\*.cs" />
     <Compile Remove="Shared\ClientsideEncryption\*.cs;Shared\ClientsideEncryption\Models\*.cs" />
+    <Compile Remove="Shared\AesGcm\**\*.cs" />
```

Comment:
All Shared classes within the Azure.Storage namespace are not included in building the Azure.Storage package. Not every class is required for every storage package. Also most of these shared classes are internal (to prevent the user from being able to call them directly), so we have to directly include the respective shared common classes in each storage package that requires it anyways.

Reply:
Ok so we haven't done the work yet to pull these into blobs and queues yet.